### PR TITLE
SigstoreSigner: convenience function for GitHub import

### LINKED ISSUE #536: SigstoreSigner: provide public key import for ambient credentials
e.g. `import_github_action()` could take repository, workflow and branch name to construct the identity string (issuer url is static):
https://github.com/secure-systems-lab/securesystemslib/blob/2acf318a9c49c0a5357e753917bd7ba5df51d1f2/.github/workflows/test-sigstore.yml#L35-L36

originally posted by @jku in #533

### PR DESCRIPTION
**Pull request by L77H**
**Description of the changes being introduced by the pull request**:
- `import_github_actions()` added that builds identity from function input and uses `https://token.actions.githubusercontent.com` for issuer. Calling `cls.import_()`.

Fixes #536 #533

### CODE REVIEW

File: securesystemslib/signer/_sigstore_signer.py [hunk 1]
```diff
@@ -268,3 +268,26 @@ def sign(self, payload: bytes) -> Signature:
...
+    @classmethod
+    def import_github_actions(
+        cls, project: str, workflow_path: str, ref: Optional[str] = "refs/heads/main"
+    ) -> Tuple[str, SigstoreKey]:
+        """Convenience method to build identity and issuer string for import_() from
+        GitHub project and workflow path.
+
+        Args:
+            project: GitHub project
+            worfklow_path: GitHub workflow path
```
Comment (jku) [inline securesystemslib/signer/_sigstore_signer.py:281]: typo fix plus added some examples to make using easier
```suggestion
            project: GitHub project name (example: "secure-systems-lab/securesystemslib") 
            workflow_path: GitHub workflow path (example: ".github/workflows/online-sign.yml")
```

**Review by jku — APPROVED** [review]
This looks right to me, thanks. 

Left a typo fix and a text change suggestion but I can take it as is too

**Comment by jku** [comment]
I did the docstring tweaks, thanks for enabling that in your branch. Merging once the CI clears

**Comment by L77H** [comment]
Great, thank you for fixing it!

**Pull request merged**